docs/content/latest/quick-start/build-apps/java/ysql-yb-jdbc.md

```diff
@@ -0,0 +1,354 @@
+    </a>
+  </li>
+  <li >
+    <a href="/latest/quick-start/build-apps/java/ysql-spring-data" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Spring Data JPA
+    </a>
+  </li>
+  <li>
+    <a href="/latest/quick-start/build-apps/java/ycql" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  <li>
+    <a href="/latest/quick-start/build-apps/java/ycql-4.6" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL (4.6)
+    </a>
+  </li>
+</ul>
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. Using the [yb-ctl](/latest/admin/yb-ctl/#root) utility, create a universe with a 3-node RF-3 cluster with some fictitious geo-locations assigned. 
+
+  ```sh
+  $ cd <path-to-yugabytedb-installation>
+
+  ./bin/yb-ctl create --rf 3 --placement_info "aws.us-west.us-west-2a,aws.us-west.us-west-2a,aws.us-west.us-west-2b"
+  ```
+
+Please note the placement values here are just tokens and have nothing to do with actual aws cloud, regions and zones.
+
+- Java Development Kit (JDK) 1.8, or later, is installed. JDK installers can be downloaded from [OpenJDK](http://jdk.java.net/).
+- [Apache Maven](https://maven.apache.org/index.html) 3.3 or later, is installed.
+
+## Create and configure the Java project
+
+1. Create a project called "DriverDemo".
+
+    ```sh
+    $ mvn archetype:generate \
+        -DgroupId=com.yugabyte \
+        -DartifactId=DriverDemo \
+        -DarchetypeArtifactId=maven-archetype-quickstart \
+        -DinteractiveMode=false
+
+    $ cd DriverDemo
+    ```
+
+1. Open the pom.xml file in a text editor and add the following below the `<url>` element.
+
+    ```xml
+    <properties>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+    ```
+
+1. Add the following dependencies for the driver HikariPool within the `<dependencies>` element in `pom.xml`.
+
+    ```xml
+    <dependency>
+      <groupId>com.yugabyte</groupId>
+      <artifactId>jdbc-yugabytedb</artifactId>
+      <version>42.3.0</version>
+    </dependency>
+
+    <!-- https://mvnrepository.com/artifact/com.zaxxer/HikariCP -->
+    <dependency>
+        <groupId>com.zaxxer</groupId>
+        <artifactId>HikariCP</artifactId>
+        <version>5.0.0</version>
+    </dependency>
+    ```
+
+1. Save and close `pom.xml`.
+
+1. Install the added dependency.
+
+    ```sh
+    $ mvn install
+    ```
+
+## Create the sample Java application
+
+You’ll create two java applications, `UniformLoadBalance` and `TopologyAwareLoadBalance`. In each, you can create connections in two ways: using the `DriverManager.getConnection()` API, or using `YBClusterAwareDataSource` and `HikariPool`. This example shows both approaches.
+
+### Uniform load balancing
+
+- Create a file called `./src/main/java/com/yugabyte/UniformLoadBalanceApp.java`.
```

Comment:
change all the bulleted steps to numbered steps - should be consistent over entire page